### ISSUE TITLE
gyb stops when trying to create label Muted

### ISSUE THREAD

**Issue by jstuker** [jay0lee/got-your-back#300]
Full steps to reproduce the issue:
1. I am restoring a mbox file created by Google Takeout to a google Gmail-account using the following command
2. ./gyb --email stuker@gmail.com --action restore-mbox --local-folder "/Users/jstuker/Downloads/Takeout"             
3. after a long (successful period) I get the following error and the cript terminats
4. Creating label Muted                                                            
5. 400: Invalid label name - invalidArgument

Workaround:

6. I try to create the label Muted in the GMail UI. Not allowed, because system label
7. I rename the label in the mbox file using: sed -i -e '/X-Gmail-Labels/s/Muted/Mutex/' All\ mail\ Including\ Spam\ and\ Trash.mbox
8. Restarting gyb works fine now

So could you skip the label creation in the code?

**Comment by Tsusai** [comment]
> 2. I rename the label in the mbox file using: sed -i -e '/X-Gmail-Labels/s/Muted/Mutex/' All\ mail\ Including\ Spam\ and\ Trash.mbox

@jstuker I ran into the same issue this weekend. I have a takeout mbox with a "muted" label, whatever that is (probably due to the new snooze/reminder options?). Thank you so much for this workaround, it's saved me from additional headaches and panic

**Comment by BobDankert** [comment]
I also ran into this issue. It was 18,000 emails into a restore-mbox action, and when I manually edited the mbox file it didn't restore properly (and I couldn't find a way to specify to skip the first 18,000 messages). To get around this and allow it to skip just the one message, I run with: `--batch-size 1 --strip-labels` and aborted the command (ctrl-c in win) after it imported the message. Then I resumed running this with the normal batch-size and without strip-labels and it continued without issues.

It would be nice if slightly better support for this could be added
